src/js/Tabs.js

```diff
@@ -56,23 +56,26 @@ function Tabs(rootEl) {
 		panelEl.setAttribute('aria-hidden', 'true');
 	}
 
-	function showPanel(panelEl) {
+	function showPanel(panelEl, focusReq) {
```

Comment:
Should optional parameters be in an args or options object?

Also, feel free to use a longer parameter name.